MatrixSDK/Crypto/KeyBackup/MXKeyBackup.m

```diff
@@ -153,7 +153,8 @@ - (void)checkAndStartWithKeyBackupVersion:(nullable MXKeyBackupVersion*)keyBacku
             Class<MXKeyBackupAlgorithm> algorithmClass = AlgorithmClassesByName[keyBackupVersion.algorithm];
             if (algorithmClass == NULL)
             {
-                MXLogError(@"[MXKeyBackup] checkAndStartWithKeyBackupVersion: unknown algorithm: %@", keyBackupVersion.algorithm);
+                NSString *message = [NSString stringWithFormat:@"[MXKeyBackup] checkAndStartWithKeyBackupVersion: unknown algorithm: %@", keyBackupVersion.algorithm];
```

Comment:
In some cases the variable is one of very few and it is okay making it a part of the message itself (objc has no `StaticString` equivalent, so this works)